mix.exs

```diff
@@ -30,8 +30,8 @@ defmodule JSONAPI.Mixfile do
   # Type `mix help deps` for more examples and options
   defp deps do
     [
-      {:phoenix, "~> 0.13"},
-      {:ecto, "~> 0.11"},
+      {:phoenix, "~> 0.14"},
```

Comment:
this is two versions behind now. We'll wanna update the whole generator to match the latest changes in that. 

This is gonna be the tricky part, keeping in sync.